.github/workflows/installer-linux-parula.yml

```
@@ -27,6 +27,8 @@ jobs:
     - run: cd e2; npm run build:release:linux
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+        NODE_OPTIONS: '--max-old-space-size=32768'
```

Could you add a comment why you do this?

```suggestion
        // For Sentry source maps
        NODE_OPTIONS: '--max-old-space-size=32768'
```